modules/apps/site-navigation/site-navigation-language-web/src/main/resources/com/liferay/site/navigation/language/web/portlet/display/template/dependencies/portlet_display_template_icon.ftl

```diff
@@ -6,6 +6,7 @@
 
 		<#if !entry.isDisabled()>
 			<@liferay_aui["icon"]
+				ariaLabel=entry.getLongDisplayName()
```

Comment:
Hey @rolandpakai, how are you going? :)

Could you provide a before and after of this change?

I would like to know what message is going to be there. I wouldn't approve this if the `ariaLabel` is the name of the symbol like `pencil`, but if this `getLongDisplayName()` returns a string containing, for example, `edit`, it will be superb

Reply:
Hi @diegonvs 
thanks, i am fine :)

In case of the Language Selector the getLongDisplayName returns the localized name of the language, 
fE: nederlands, français, deutsch, magyar, español

Thanks, Roland

Reply:
Nice then! Thanks, Roland! :D